### ISSUE TITLE
Error on authenticating

### ISSUE THREAD

**Issue by Tjoosten** [KnpLabs/php-github-api#775]
Hi I've experienced an issue over the authentication on github trough the api. 

**Error:**

```
Too few arguments to function Http\Adapter\Guzzle6\Client::buildClient(), 0 passed in C:\Users\Gebruiker\Documents\portal-nuc\vendor\php-http\guzzle6-adapter\src\Client.php on line 31 and exactly 1 expected
```

**code**

```
$githubClient = new \Github\Client(); 
        $credentials  = [config('github.user'), config('github.password'), \Github\Client::AUTH_HTTP_PASSWORD];

        dd($githubClient->authenticate($credentials));
        die();
```

Is this package related or is the fault on my end? 

Kind Regards, 
Tim

**Comment by mattgrayisok** [comment]
This is caused by an issue in guzzle6-adapter which was fixed here:

https://github.com/php-http/guzzle6-adapter/commit/594cd47fb9af5c217f0e63c4e8f7602da88a368a

The fix was released in v1.1.1 however the install instructions for php-github-api define v1.1 explicitly.

Instead of `composer require knplabs/github-api php-http/guzzle6-adapter "^1.1"` use `composer require knplabs/github-api php-http/guzzle6-adapter "~1.1"`

**Comment by acrobat** [comment]
Thanks for the feedback @mattgrayisok! Our readme correctly states the `^1.1` constraint for the guzzle6-adapter so I'm closing this issue, but your comment can help others having this issue and searching our issue tracker!
